checkov/common/bridgecrew/platform_integration.py

```diff
@@ -343,6 +347,8 @@ def persist_repository(
                 for file_path in f_names:
                     _, file_extension = os.path.splitext(file_path)
                     if file_extension in SUPPORTED_FILE_EXTENSIONS or file_path in SUPPORTED_FILES:
+                        if RUN_NEW_SCA_PACKAGE_SCAN and file_extension in SUPPORTED_PACKAGE_FILES:
```

Comment:
this function is to persist the files.
If we don't want to persist the files in case the new sca package scan.
I think we should no call this function at all in that case.

Reply:
we do not persisting the package files here because they already uploaded in the scanner